samples/client/src/main/java/global/covesa/sdk/client/push/ActionEvent.kt

```diff
@@ -0,0 +1,41 @@
+package global.covesa.sdk.client.push
+
+import android.app.Activity
+import global.covesa.sdk.api.client.push.PushManager
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+
+class ActionEvent(private val type: Type) {
+    enum class Type {
+        RegisterPush,
+        UnregisterPush,
+        SendNotification,
+    }
+
+    fun handleAction(activity: Activity) {
+        when(type) {
+            Type.RegisterPush -> registerPush(activity)
+            Type.UnregisterPush -> PushManager.unregister(activity)
+            Type.SendNotification -> MockApplicationServer(activity).MockApi().sendNotification()
+        }
+    }
+
+    private fun registerPush(activity: Activity) {
+        PushManager.tryUseCurrentOrDefaultDistributor(
+            activity
+        ) {
+            PushManager.register(
```

Comment:
Aren't you supposed to check the `success` argument before calling `register()`? Also, this API seems redundant. 1 call to `PushManager` should be sufficient (see my `PushManager` comments).